HttpSignature.md

```diff
@@ -223,11 +236,17 @@ App                                   Server                Doc             Cred
 
 Steps (4) and (5), where the server retrives a (cached)copy of the key, are as before. 
 
-Steps (6) can be run in parallel with (4) to fetch the Credential document. This also can be cached. If the URL is relative it will be found on Resource Server. If the URL is remote it may need to be fetched. If the URL is enclosed in `>` and `<` and a P2P extension is available the server can request the credential from the client directly using the same connection opened in (3).
+Steps (6) can be run in parallel with (4) to fetch the Credential document.
+This also can be cached.
+If the URL is relative it will be found on Resource Server.
+If the URL is remote it may need to be fetched.
+If the URL is enclosed in `>` and `<` and a P2P extension is available the server can request the credential from the client directly using the same connection opened in (3).
```

Comment:
This is unclear to me. Is the enclosure described `>http://example.com/example<` or `<http://example.com/example>`?  (In either case, the comma inserted below is suggested.)
```suggestion
If the URL is enclosed in `>` and `<` and a P2P extension is available, the server can request the credential from the client directly using the same connection opened in (3).
```

Reply:
yes, the idea is to have >/creds/key1< as keys. I'll add there that relative URLs are meant, as the server may know nothing about the client other than that it is at the other end of the connection.

Reply:
Actually I am not sure if absolute URLs may not be useful there.

Reply:
I tried to clarify that point and linked to an interesting did related idea I put up on Solid.